EVA-3759 Update Submit Page

nitin-ebi opened the pull request:
Changes deployed in dev : [https://wwwdev.ebi.ac.uk/eva/EVA-3759](https://wwwdev.ebi.ac.uk/eva/EVA-3759/?Submit-Data)

tcezard commented on src/index.html:
```diff
@@ -291,26 +280,71 @@ <h3 id="key-stages" class="padding-bottom-large">Key stages of EVA submissions</
+                                <div class="box blue">
+                                    <h3>✉️ Submit via email</h3>
+                                    <p>Contact <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a> to request a submission. You will receive a custom private FTP account to deposit your data and metadata files, as well as further instructions.</p>
+                                    <p>This submission method is less technical but may involve longer processing times.</p>
```
```suggestion
                                    <p>This submission method is less technical but may involve longer processing times as we help you validate your data.</p>

```

nitin-ebi replied:
updated

tcezard commented on src/index.html:
```diff
@@ -291,26 +280,71 @@ <h3 id="key-stages" class="padding-bottom-large">Key stages of EVA submissions</
+                                <p>All data that is valid for EVA submission shall be made available via the <a href="?Study Browser" target="_blank">Study Browser</a> and will be browsable using both the <a href="?Variant Browser" target="_blank">Variant Browser</a> and the <a href="?API" target="_blank">EVA API</a>, and will also be incorporated into our periodic <a href="?RS-Release" target="_ ...
+                            </div>
+                        </div>
+                        <p>There are two methods of submitting to EVA:</p>
+                        <ol>
+                            <li>
+                                <p><b>Automated command line (faster)</b></p>
+                                <p>The first is <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>, a command-line tool that both validates your data and metadata against EVA requirements, and completes the submission by uploading your data and metadata to EVA servers. This is more comprehensive than the vcf-validator and provides faster processing times than su ...
+                                <p>For instructions on how to install and use eva-sub-cli, please see the documentation on <a href="https://github.com/EBIvariation/eva-sub-cli/blob/main/README.md" target="_blank">Github</a>. This tool is new and being constantly improved, so if you have feedback or bug reports please let us know.</p>
+                            </li>
+                            <li>
+                                <p><b>Submission via helpdesk (slower)</b></p>
+                                <p>The other method of submission is by contacting <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a> to request a submission. You will receive a custom private FTP account to deposit your data and metadata files, as well as further instructions. This method is more manual and may involve slower processing times, but it may be suitable ...
+                                <p>Note that if your data includes <b>individual human genotype data</b>, then we may request a <a href="https://docs.google.com/document/d/1UaRmimAe919IZYIH55mDXMe-4sHsRVqaP4moqx2IYE4/edit" target="_blank">consent statement</a> which must be completed prior to archival.</p>
+                            </li>
+                        </ol>
```
I don't think we need these section either.

tcezard reviewed (commented):
We can discuss if we need the text section that repeat what is in the diagram. but We should put the diagram first.

nitin-ebi commented on src/index.html:
```diff
@@ -252,36 +252,25 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
-                                <b><u>PLEASE NOTE:</u></b> Sequence identifiers in VCF must match those in the reference FASTA file.
-                            </li>
-                            <li>If consent was gathered for any <b>individual human genotype data</b> then a <a href="https://docs.google.com/document/d/1UaRmimAe919IZYIH55mDXMe-4sHsRVqaP4moqx2IYE4/edit" target="_blank">consent statement</a> must be completed prior to submission.</p></li>
+                            <li>Data is described in <b>valid VCF file(s)</b>. This can be tested prior to submission using the <a href="https://github.com/EBIvariation/vcf-validator/wiki/User's-Guide" target="_blank">EVA VCF validation suite</a> or <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli tool</a>. For help with converting variation data to VCF, please se ...
```
removed

nitin-ebi commented on src/index.html:
```diff
@@ -252,36 +252,25 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
+                            <li>Data is described in <b>valid VCF file(s)</b>. This can be tested prior to submission using the <a href="https://github.com/EBIvariation/vcf-validator/wiki/User's-Guide" target="_blank">EVA VCF validation suite</a> or <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli tool</a>. For help with converting variation data to VCF, please se ...
+                            <li>Data includes <b>sample genotypes</b> in the FORMAT fields and/or <b>allele frequencies</b> in the INFO field</li>
+                            <li>The reference sequence used is <a href="http://www.insdc.org/" target="_blank">INSDC</a> registered, or will be at point of submission. The type of sequence should be an assembly, e.g. <a href="https://www.ebi.ac.uk/ena/browser/view/GCA_000002285.4" target="_blank">GCA_000002285.4</a></li>
+                            <li><b><u>PLEASE NOTE:</u></b> We support multiple sequence identifiers (CHROM column in VCF), such as INSDC or Refseq identifiers and ENA and Genbank chromosome names. For more details, please see our <a href="?Help#submissionPanel&link=how-to-gen-vcf" target="_blank">help</a> pages.</li>
```
corrected

nitin-ebi commented on src/index.html:
```diff
@@ -291,26 +280,71 @@ <h3 id="key-stages" class="padding-bottom-large">Key stages of EVA submissions</
+                                <h3>📄 Prepare your data and metadata</h3>
+                                <ul>
+                                    <li>Prepare valid VCF file(s), which can be validated prior to submission using the <a href="https://github.com/EBIvariation/vcf-validator/wiki/User's-Guide" target="_blank">EVA VCF validation suite</a> or the <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli tool</a>.</li>
+                                    <li>Complete a <a href="https://www.ebi.ac.uk/eva/files/EVA_Submission_template.V1.1.5.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. This can be validated alongside your VCF files with <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>. Please provide as much metadata as possibl ...
```
updated

nitin-ebi commented on src/index.html:
```diff
@@ -895,6 +931,22 @@ <h5>Archive process</h5>
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>Which data is brokered to DBSNP?</h5></a>
+                                        <div class="accordion-content" data-tab-content>
+                                            <p>Human data submitted to EVA will be brokered to DBSNP</p>
```
updated

nitin-ebi commented on src/index.html:
```diff
@@ -895,6 +931,22 @@ <h5>Archive process</h5>
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>How do we deal with Chromosome name diversity?</h5></a>
+                                        <div class="accordion-content" data-tab-content>
+                                            <p>We support multiple sequence identifiers for Chromosome name, such as INSDC or Refseq identifiers and ENA and Genbank chromosome names.</p>
```
updated

nitin-ebi commented:
> We can discuss if we need the text section that repeat what is in the diagram. but We should put the diagram first.

Moved the diagram up and removed the text section

apriltuesday commented on src/index.html:
```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
+                            <h3>📄 Prepare your data and metadata</h3>
+                            <ul>
+                                <li>Prepare valid VCF file(s), which can be validated prior to submission using the <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli tool</a>.</li>
+                                <li>Complete a <a href="https://raw.githubusercontent.com/EBIvariation/eva-sub-cli/main/eva_sub_cli/etc/EVA_Submission_template.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. This can be validated alongside your VCF files with <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>. Pleas ...
```
```suggestion
                                <li>Complete a <a href="https://raw.githubusercontent.com/EBIvariation/eva-sub-cli/main/eva_sub_cli/etc/EVA_Submission_template.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. This can be validated alongside your VCF files with <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>. Please provide as much metadata as possible since this information is useful for downstream analysis and leads to increased dataset reuse. For reference, here is an <a href="https://raw.githubusercontent.com/EBIvariation/eva-sub-cli/main/eva_sub_cli/etc/EVA_Submission_Example.xlsx" target="_blank">example</a> of a completed metadata template.</li>

```
This is the updated example. Also not sure it should be bold since nothing else in the box is...

nitin-ebi replied:
updated.

apriltuesday commented on src/index.html:
```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
-                        <p>Upload your VCF file(s), metadata template and any associated data file(s) to your private FTP location.</p>
+                    <div class="submit-data-diagram">
+                        <div class="box green">
+                            <h3>📄 Prepare your data and metadata</h3>
```
Personally I don't think the emojis look professional, I would use the same icon source that is used in the current submit page - we just need to find the appropriate additional icons for the new steps. If necessary, we can use the icons in the document diagram, just need to credit [flaticon.com]() somewhere as that's the source I used.

apriltuesday commented on src/index.html:
```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
+                                <p>This method has faster turnaround times, but may be more suitable for more technical users.</p>
+                                <p>Before submitting data through eva-sub-cli, please make sure you have a <a href="https://www.ebi.ac.uk/ena/submit/webin/login" target="_blank">Webin</a> Account.</p>
+                            </div>
+                            <div class="or-label">OR</div>
```
Should the OR be a bit larger? Maybe as large as the titles in the boxes

nitin-ebi replied:
updated

apriltuesday commented on src/index.html:
```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
+                                <h3>✉️ Submit via email</h3>
+                                <p>Contact <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a> to request a submission. You will receive a custom private FTP account to deposit your data and metadata files, as well as further instructions.</p>
+                                <p>This submission method is less technical but may involve longer processing times as we help you validate your data.</p>
+                                <p>Please make sure to read the instructions given in the metadata spreadsheet before submitting.</p>
```
I feel like the last point should go in the Prepare box, since it really applies to both paths

nitin-ebi replied:
As discussed in the standup, leaving as it is for now

apriltuesday commented on src/index.html:
```diff
@@ -895,6 +896,22 @@ <h5>Archive process</h5>
+                                </div>
+                                <div id="how-do-we-deal-with-chromosome-name-diversity">
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>How do we deal with Chromosome name diversity?</h5></a>
```
Maybe:
```suggestion
                                        <a class="accordion-title"><h5>How do we deal with chromosome or contig name diversity?</h5></a>

```
Just because we kind of use both terms interchangeably in the text...

nitin-ebi replied:
updated

apriltuesday commented on src/index.html:
```diff
@@ -895,6 +896,22 @@ <h5>Archive process</h5>
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>How do we deal with Chromosome name diversity?</h5></a>
+                                        <div class="accordion-content" data-tab-content>
+                                            <p>Contig naming convention can vary depending on how the reference genome was obtained. We strive to support multiple chromosome name such as INSDC or Refseq identifiers and ENA and Genbank chromosome names. In case we do not support the naming convention you have used please contact eva-helpdesk@ebi.ac.uk.</p>
```
```suggestion
                                            <p>Contig naming convention can vary depending on how the reference genome was obtained. We strive to support multiple chromosome names, such as INSDC or RefSeq identifiers and ENA and Genbank chromosome names. In case we do not support the naming convention you have used, please contact <a href="mailto:eva-helpdesk@ebi.ac.uk">eva-helpdesk@ebi.ac.uk</a>.</p>

```

nitin-ebi replied:
updated

apriltuesday reviewed (commented):
I think it looks really good! Personally I don't miss the text-only "key stages" at all.

nitin-ebi commented on src/index.html:
```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
+                            <li>The reference sequence used is <a href="http://www.insdc.org/" target="_blank">INSDC</a> registered, or will be at point of submission. The type of sequence should be an assembly, e.g. <a href="https://www.ebi.ac.uk/ena/browser/view/GCA_000002285.4" target="_blank">GCA_000002285.4</a></li>
+                            <li><b><u>PLEASE NOTE:</u></b> We support multiple sequence identifiers (CHROM column in VCF), such as INSDC or Refseq identifiers and ENA and Genbank chromosome names. For more details, please see our <a href="?Help#submissionPanel&link=how-do-we-deal-with-chromosome-name-diversity" target="_blank">help</a> pages.</li>
                         </ol>
                         <p><a href="?Help#accessionPanel&link=variant-accession-administred-by-eva" target="_blank">Variant accessions</a> (ss# and rs#) and <a href="?Help#accessionPanel&link=accessions-administred-by-eva" target="_blank">study accessions</a> will only be provided for data which satisfies all data requirements. More details on whether your data is suitable can be found <a href="? ...
```
removed
